### PR TITLE
Add release v.4.8.0.5

### PR DESCRIPTION
**Pull request by Moviss**
Updated index.html due to new releases info.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -801,6 +872,56 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.
...
+<div class="dimmed-text"><p>Release date: 2018-09-28</p></div>
+
+<ul>
+    <li>Fixed/added:</li>
+    <ol>
+        <li>Updated sortbootorder to v4.6.11</li>
+    </ol>
+    <li>Known issues:</li>
+    <ol>
+        <li>pfSense 2.4.x fails to boot when no USB stick is plugged</li>
```
Comment (miczyg1) [inline index.html]: @Moviss I tested v4.0.20 and had no reproduction of the pfSense issue. This should be removed.

File: index.html [hunk 2]
```diff
@@ -801,6 +872,56 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.
...
+            <li><a href="http://pcengines.ch/file/apu4_v4.0.20.rom.tar.gz">apu4 v4.0.20</a></li>
+            <li><a href="http://pcengines.ch/file/apu5_v4.0.20.rom.tar.gz">apu5 v4.0.20</a></li>
+        </ul>
+    </div>
+    <div class="source-code-column">
+        <div class="smaller-margin"><p>Source code:</p></div>
+        <ul>
+            <li><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.0.19...v4.0.20">coreboot v4.0.20</a></li>
+            <li><a target="_blank" href="https://github.com/pcengines/seabios/compare/rel-1.11.0.4...rel-1.11.0.5">SeaBIOS rel-1.11.0.5</a></li>
+            <li><a target="_blank" href="https://github.com/pcengines/sortbootorder/compare/v4.6.5...v4.6.9">sortbootorder v4.6.9</a></li>
```
Comment (miczyg1) [inline index.html]: @Moviss the link to sortbootorder compare should be
`https://github.com/pcengines/sortbootorder/compare/v4.6.9...v4.6.11`

Comment (Moviss) [reply index.html]: @miczyg1 a tekst powinie być analogicznie: `sortbootorder v4.6.11`?

File: index.html [hunk 3]
```diff
@@ -125,6 +127,66 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
...
+
+    <div class="dimmed-text"><p>Release date: 2018-10-04</p></div>
+
+    <ul>
+        <li>Fixed/added:</li>
+        <ol>
+            <li>Updated sortbootorder to v4.6.11</li>
+            <li>Removed custom libpayload modifications</li>
+            <li>Synced and rebased with coreboot official repo commit b7b085d</li>
+            <li>ECC exclusion range, ECC now works properly</li>
```
Comment (miczyg1) [inline index.html]: @pietrushnic maybe we should link the blog post here. What do You think?
@Moviss FYI

Comment (miczyg1) [reply index.html]: @pietrushnic @Moviss ?

**Pull request merged**